projects/plutus/test/test_current_calibration.c

```diff
@@ -0,0 +1,60 @@
+#include <inttypes.h>
```

Comment:
How does this test work?

Reply:
It works similarly to the throttle calibration test. There is a delay so that the user can set the required current, and then the module samples a predefined number of adc readings in order to generate the data points needed for calibration.

Reply:
Please add a comment explaining this is not actually a test, but is actually a calibration routine.

Based on random samples of the ADC, you may want to add an additional offset at 0A. It seems like the 0 point changes every time the chip is powered. That can be zeroed by the application (we know that when the relays are open, no current is flowing).

Reply:
ping